src/App.scss

```diff
@@ -0,0 +1,62 @@
+$main-font: 'Stardos Stencil', cursive;
```

Comment:
Oh, so really the only difference is that you change css to scss on the file path? Dope!

Reply:
Yeah. You also have to install Sass as a dependency with the command below: 
`npm install node-sass --save`